src/applications/yellow-ribbon/helpers/index.js

```diff
@@ -13,3 +14,30 @@ export const normalizeResponse = response => ({
   })),
   totalResults: response?.meta?.count,
 });
+
+export const updateQueryParams = (
+  history = window.history,
+  location = window.location,
+  queryParamsLookup = {},
+) => {
+  // Derive the current query params.
+  const queryParams = new URLSearchParams(location.search);
+
+  // Set/Delete query params.
+  Object.keys(queryParamsLookup).forEach(key => {
+    // Derive the value.
+    const value = queryParamsLookup[key];
```

Comment:
[Object.entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) will get you both the key/value